tests/lib/rules/no-dupe-keys.js

```diff
@@ -22,6 +22,11 @@ ruleTester.run("no-dupe-keys", rule, {
     valid: [
         "var foo = { __proto__: 1, two: 2};",
         "var x = { foo: 1, bar: 2 };",
+        "var x = { '': 1, bar: 2 };",
+        "var x = { '': 1, ' ': 2 };",
+        { code: "var x = { '': 1, [null]: 2 };", parserOptions: { ecmaVersion: 6 } },
+        { code: "var x = { '': 1, [a]: 2 };", parserOptions: { ecmaVersion: 6 } },
+        { code: "var x = { [a]: 1, [a]: 2 };", parserOptions: { ecmaVersion: 6 } },
```

Comment:
I'm probably missing something, but I'm curious why the rule doesn't warn on this.

Reply:
Maybe by design. Or perhaps the rule was made when there were no computed properties in obj literals. I'll open an issue for this enhancement

Reply:
Compare computed keys by tokens in no-dupe-keys #12070

Reply:
As discussed in #12070, it doesn't seem like we can reliably and safely enforce this case.